### PR TITLE
Updates to docstrings for preview 5.

### PR DESCRIPTION
**Pull request by annelo-msft**

### CODE REVIEW

**Review by AlexGhiondea — APPROVED** [review]

File: sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs [hunk 1]
```diff
@@ -39,7 +39,7 @@ public ConfigurationClient(string connectionString)
         /// Creates a <see cref="ConfigurationClient"/> that sends requests to the configuration store.
```
Comment (heaths) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs]: Nit, though you are consistent in your library: standard .NET text is something like `Initializes an instance of the <see cref="ConfigurationClient"/> class"` - at least as a prefix (so you could add things like "for mocking.", etc. Not saying you should change it, but it's the standard text that .NET itself uses. Key Vault and Storage, IIRC, use this form as well.

File: sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs [hunk 2]
```diff
@@ -618,10 +604,11 @@ internal virtual Response<ConfigurationSetting> GetConfigurationSetting(string k
         }
 
         /// <summary>
-        /// Retrieves one or more <see cref="ConfigurationSetting"/> that satisfies the options of the <see cref="SettingSelector"/>
+        /// Retrieves one or more <see cref="ConfigurationSetting"/> that satisfy the options set in the <see cref="SettingSelector"/>.
         /// </summary>
-        /// <param name="selector">Set of options for selecting <see cref="ConfigurationSetting"/> from the configuration store.</param>
+        /// <param name="selector">Options used to select a set of <see cref="ConfigurationSetting"/>s from the configuration store.</param>
```
Comment (heaths) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs]: Tip: I used to add an "s" on the end of type/member references, but it often comes through looking weird, like:

> `ConfigurationSetting`s.

What I've been doing instead is something like:

> `ConfigurationSettings` values

File: sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientOptions.cs [hunk 3]
```diff
@@ -18,13 +18,13 @@ public class ConfigurationClientOptions : ClientOptions
         internal const ServiceVersion LatestVersion = ServiceVersion.V1_0;
 
         /// <summary>
-        /// The versions of App Config Service supported by this client library.
+        /// The versions of the App Configuration service supported by this client library.
```
Comment (heaths) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientOptions.cs]: Does Azure refer to it as "App Configuration" or "Application Configuration"? Might want to match its text.

File: sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationSetting.cs [hunk 4]
```diff
@@ -35,19 +34,19 @@ public ConfigurationSetting(string key, string value, string label = null)
         }
 
         /// <summary>
-        /// The primary identifier of a configuration setting.
-        /// The key is used in unison with the label to uniquely identify a configuration setting.
+        /// The primary identifier of the configuration setting.
+        /// The key is used together with label to uniquely identify a configuration setting.
```
Comment (heaths) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationSetting.cs]: Here and just below, I would say "a label" or "a key"; or even refer to them via `<see cref=""/>` for a nice tie between the doc pages.

File: sdk/appconfiguration/Azure.Data.AppConfiguration/src/SettingSelector.cs [hunk 5]
```diff
@@ -4,49 +4,63 @@
...
 
 namespace Azure.Data.AppConfiguration
 {
     /// <summary>
-    /// Set of options for selecting <see cref="ConfigurationSetting"/> from the configuration store.
+    /// <see cref="SettingSelector"/> is a set of options that allows selecting
+    /// a filtered set of <see cref="ConfigurationSetting"/>s from the
+    /// configuration store, and optionally allows indicating which fields of
+    /// each setting to retreive.
+    ///
```
Comment (heaths) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/src/SettingSelector.cs]: These blank lines will be canonicalized into one long paragraph. If you want to keep them, put each block in `<para></para>` tags.

File: sdk/appconfiguration/Azure.Data.AppConfiguration/src/SettingSelector.cs [hunk 6]
```diff
@@ -4,49 +4,63 @@
...
 using System.ComponentModel;
-using System.Linq;
-using Azure.Core;
 
 namespace Azure.Data.AppConfiguration
 {
     /// <summary>
-    /// Set of options for selecting <see cref="ConfigurationSetting"/> from the configuration store.
+    /// <see cref="SettingSelector"/> is a set of options that allows selecting
+    /// a filtered set of <see cref="ConfigurationSetting"/>s from the
```
Comment (heaths) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/src/SettingSelector.cs]: If you change to not append the "s", I see a few instances in this file as well.

**Review by heaths — APPROVED** [review]

File: sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientOptions.cs (hunk 3 above)
Comment (annelo-msft) [reply sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientOptions.cs]: It's App Configuration - https://docs.microsoft.com/en-us/azure/azure-app-configuration/overview

**Pull request merged**